packages/logos-docusaurus-theme/src/client/components/mdx/TimelineItem/TimelineItem.tsx

```diff
@@ -75,6 +100,8 @@ export const TimelineItem: React.FC<TimelineItemProps> = ({
   children,
   ...props
 }) => {
+  const isTransparent = isFuture(period)
+
```

Comment:
@jinhojang6, I find this inconsistent with how other properties affect the component's behavior. For instance, `borderStyle` should be decided based on the other properties as well, but currently, we let it to be explicitly set to either `dashed`, `solid`, or `none`. Also, we currently let users pass anything as `period`, the property's type is `React.ReactNode` and not `string`, so I don't think it's a good idea to parse it.

Since this component is getting complicated, I suggest:
1. keep this component flexible and add a new property `periodStyle` with `transparent` and `filled` as options.

2. Restore the `Timeline` component and handle the logic there:
- `borderStyle`:
  - `solid` for events in the same year
  - `none` for the last event
  - `dashed` for the rest
- `periodStyle`:
   - `transparent` for the future events
   - `filled` for the rest

3. In the `Timeline` component let setting `period` for each item with a different and more explicit type: `[number, 'Q1' | 'Q2' | 'Q3' | 'Q4' | '+'] | number` -  e.g., `2023` or `[2023, 'Q1']` or `[2024, '+']`